ElevatorSim/src/elevatorsim/floor/FloorController.java

```diff
@@ -20,11 +15,17 @@
 
 public class FloorController extends Thread implements MessageReciever {
 	private HashMap<Integer, Floor> floors;
-	private List<MessageRequest> requests;
+	private Map<Integer, MessageRequest> requests;
 	
-	public FloorController(String name, HashMap<Integer, Floor> floors, List<MessageRequest> requests) {
+	public FloorController(String name, int numOfFloors, Map<Integer, MessageRequest> requests) {
```

Comment:
I modified this constructor. I thought floor construction to be in the domain of the Floor Controller rather than that of main.

Reply:
The change looks good.